lib/fx/adapters/postgres.rb

```diff
@@ -125,7 +125,11 @@ def update_trigger(name, on:, sql_definition:)
       #
       # @return [void]
       def drop_function(name)
-        execute "DROP FUNCTION #{name}();"
+        fns = functions.select { |function| function.name == name.to_s }
+
+        fns.each do |function|
+          execute "DROP FUNCTION #{name}(#{function.arguments});"
```

Comment:
What do you think introducing a `Function#signature` method? That way we could return `test()` or `test(int a)` from that method, and clarify this sql exectution?